Sources/StreamChat/WebSocketClient/RetryStrategy.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// The type encapsulating the logic of computing delays for the failed actions that needs to be retried.
+protocol RetryStrategy {
+    /// Rerturns the # of consecutively failed retries.
+    var consecutiveFailures: Int { get }
```

Comment:
```suggestion
    var consecutiveFailuresCount: Int { get }
```

Could we attach count/attempts suffix here for better readability? Thanks

Reply:
✅ Updated